cli/cmd/release/prepare.go

```diff
@@ -0,0 +1,36 @@
+package release
+
+import (
+	"github.com/spf13/cobra"
+	"github.com/wordpress-mobile/gbm-cli/pkg/console"
+	"github.com/wordpress-mobile/gbm-cli/pkg/release"
+	"github.com/wordpress-mobile/gbm-cli/pkg/utils"
+)
+
+var PrepareCmd = &cobra.Command{
+	Use:   "prepare",
+	Short: "Prepare a release",
+	Long:  `Use this command to prepare a release`,
+	Run: func(cmd *cobra.Command, args []string) {
+		version, err := getVersionArg(args)
+		console.ExitIfError(err)
+
+		console.Info("Preparing release for version %s", version)
+
+		tempDir, err := utils.SetTempDir()
+		console.ExitIfError(err)
+
+		defer utils.CleanupTempDir(tempDir)
+
+		console.Info("Created temporary directory %s", tempDir)
+
+		pr, err := release.CreateGbPR(version, tempDir)
+		console.ExitIfError(err)
+
+		console.Info("Created PR  (url ? %s)", pr)
```

Comment:
The output of this command seemed like it might not be quite right when I ran this. Some values from the API appear to be missing, and the formatting does not seem like it's using the correct line breaks. I think this may be because I did not set up the API auth token correctly for my own personal GitHub organization when testing, which results in some of the missing or nil values in the response. Just posting this to see if your output here from this command is any different. Here's the text of the terminal output (and a screenshot from the terminal to check the syntax highlighting, if that helps):

```
To github.com:derekblank/gutenberg
 * [new branch]      rnmobile/release_1.0.1 -> rnmobile/release_1.0.1
PR created: &{%!!(MISSING)s(int=2) https://github.com/derekblank/gutenberg/pull/2 https://api.github.com/repos/derekblank/gutenberg/pulls/2 ## Description
Release 1.0.1 of the react-native-editor and Gutenberg-Mobile.

 Mobile Release v1.0.1 [{Mobile App - i.e. Android or iOS}] open {derekblank} %!!(MISSING)s(bool=false) %!!(MISSING)s(bool=false) {rnmobile/release_1.0.1 f755a74fe3b45194f73d95a724bb9a89924fde98 {}} {trunk 416bab68238be99fa77429d9d6a1e0174fdd9507 {}} []  }

Created PR  (url ? {%!!(MISSING)s(int=2) https://github.com/derekblank/gutenberg/pull/2 https://api.github.com/repos/derekblank/gutenberg/pulls/2 ## Description
Release 1.0.1 of the react-native-editor and Gutenberg-Mobile.

 Mobile Release v1.0.1 [{Mobile App - i.e. Android or iOS}] open {derekblank} %!!(MISSING)s(bool=false) %!!(MISSING)s(bool=false) {rnmobile/release_1.0.1 f755a74fe3b45194f73d95a724bb9a89924fde98 {}} {trunk 416bab68238be99fa77429d9d6a1e0174fdd9507 {}} []  })
```

<img width="1416" alt="Screenshot 2023-10-10 at 4 32 12 pm" src="https://github.com/wordpress-mobile/release-toolkit-gutenberg-mobile/assets/643285/f9659b59-4846-42fa-bb25-8e08f01fa79b">

Reply:
It looks like the [PR was created ](https://github.com/derekblank/gutenberg/pull/2)but it didn't add labels 🤔 
That might be a GH auth issue but seems odd that you could only create the PR but not the labels.

I think the odd output has more to do with printing the `gh.PullRequest`. The gh api request places the response in the `gh.PullRequest` struct but I wonder if those missing values relate to properties in the response interface which are not in the struct.

Reply:
Here is a bit more on the `MISSING` stuff https://www.reddit.com/r/golang/comments/j7ryfd/why_does_a_missing_argument_or_type_mismatch_in/

Reply:
Just realized why the labels were not created.
The labels have to exist on the repo before adding them to a PR. 
I'm open on how to handle this scenario. The command should least make it clear that the labels were not present.  The issue is that with the current GH api we have to add the labels after we create the PR, so if we want to fail the command we should first check if the label exists before creating the PR.

Given that the labels are hard coded to match the main repos I think it's a bit overkill to do the label check. I think just making it very clear warning that the labels failed should be enough. Perhaps warn with "Adding the labels failed, are you sure the label {label} exists on the repo?"

Reply:
> The labels have to exist on the repo before adding them to a PR.
That makes sense! 

> Given that the labels are hard coded to match the main repos I think it's a bit overkill to do the label check. I think just making it very clear warning that the labels failed should be enough.
I think this is plenty. The labels will already exist on the GB and GBM repos, so the chance of encountering this would be very minimal in production. Plus, it's trivial to add the labels manually.